.gitlab/kernel_version_testing/system_probe.yml

```diff
@@ -32,8 +32,6 @@ pull_test_dockers_arm64:
 .package_dependencies:
   stage: kernel_matrix_testing
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/system-probe_x64$DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX:$DATADOG_AGENT_SYSPROBE_BUILDIMAGES
-  rules:
```

Comment:
I think we still need `rules` of some sort, so they don't run on every pipeline, right?